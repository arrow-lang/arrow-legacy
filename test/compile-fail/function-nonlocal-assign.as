let a;

def method() {
  a = 30; // error
}
