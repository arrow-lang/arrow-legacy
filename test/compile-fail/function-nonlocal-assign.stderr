[0;37mtest/compile-fail/function-nonlocal-assign.as:4.3-4: [0;31merror: [1;37mcannot assign immutable non-local variable `a`[0m
